arch/s390/Makefile.in

```diff
@@ -30,6 +30,16 @@ dfltcc_inflate.o:
 dfltcc_inflate.lo:
 	$(CC) $(SFLAGS) $(INCLUDES) -c -o $@ $(SRCDIR)/dfltcc_inflate.c
 
+VX_CRC32_FLAGS:=-march=z13
+VX_CRC32_FLAGS:=$(VX_CRC32_FLAGS) $(shell $(CC) -x c -c /dev/null -mzarch 2>/dev/null && echo -mzarch)
+VX_CRC32_FLAGS:=$(VX_CRC32_FLAGS) $(shell $(CC) -x c -c /dev/null -fzvector 2>/dev/null && echo -fzvector)
```

Comment:
All variables should be filled by `configure` or `CMakeLists.txt` as `Makefile.in` is ignored by cmake.

Reply:
`CMakeLists.txt` already has its own version of this code - this here is just for `configure`-based processing. I didn't put this into `configure`, because it's needed for s390 only, but I can do that if you think that would be more maintainable.

Reply:
configure has one function per architecture, just like cmake... that makes it quite isolated and clean.